src/components/Main.vue

```diff
@@ -0,0 +1,211 @@
+                                        class="textarea text-input"
+                                        placeholder="Paste your letter of recommendation here."
+                                    ></textarea>
+                                    <div class="btn-container">
+                                        <button
+                                            class="button is-info is-fullwidth submit-button"
+                                            v-on:click="renderIssues" 
+                                            @click = "showSidebar(); hideHeader(); changeWidth()"
+                                        >
+                                            Submit
+                                        </button>
+                                    </div>
+                                </div>
+                                <div v-if="rendered">
+                                    <button
+                                        class="button is-primary is-fullwidth back-button"
+                                        v-on:click="
+                                            () => {
+                                            rendered = false;
+                                            }
+                                        "
+                                    >
+                                        &leftarrow; Again!
+                                    </button>
+                                    <div class = 'blurbs' v-bind:style="{width: widthVal}">
+                                        <Blurb
+                                        v-for="message in messages"
+                                        :key="message.rnd"
+                                        :message="message"
+                                        />
+                                    </div>
+                                </div>
+                            </div>
+                        </div> 
+                        <div class ='sidebar_container' v-if = 'sidebarStatus'>
+                            <div class = 'sidebar'> 
+                                <p> Try to make the following changes: </p> 
+                                <Summary v-for="summary in summaries"
+                                 :key= "summary.rnd"
+                                 :summary ="summary" />
+                            </div> 
+                        </div>
+                    </span> 
+                </div>
+            </div>
+        </div> 
+    </div>
+    
+</template>
+<script>
+import Blurb from "./Blurb";
+import Summary from "./Summary.vue"
+const URL = "http://localhost:5000";
+export default {
+    name: "Main",
+    components: {
+        Blurb,
+        Summary
+    },
+    data() {
+        return {
+            text: "",
+            inputText: "",
+            messages: [],
+            summaries: [],
+            rendered: false,
+            sidebarStatus: false,
+            widthVal: '700px'
```

Comment:
I'm a little nervous about setting a fixed pixel width here. That means the layout is no longer responsive. What's the reason for it?

Reply:
@ahmed and I (Zoë also worked on part of this) had a really hard time keeping the textbox the right size and having it be smaller only after pressing submit and we ended up fixing the width.